Docs: Add a callout about accessibility on tooltips hiding important content (#2021)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.3/components/tooltips.md

Reviewer: TBH, I'd put this either at the very top of the page or right after the two info callouts coming from Bootstrap in `## Overview`

Reviewer: Yeah OK, let's see what's the pov of Julien!

Reviewer 2: Maybe you can do it like in the [Dropdowns](https://deploy-preview-2021--boosted.netlify.app/docs/5.3/components/dropdowns/). Overview, then Accessibility, then Examples.

Reviewer 2: Linked to my previous comment, there's an issue in our Tooltips docs (https://twbs-bootstrap.netlify.app/docs/5.3/components/tooltips/). **Examples** shouldn't be at the same level as **Overview** (see https://twbs-bootstrap.netlify.app/docs/5.3/components/tooltips/).

So you could probably have:
1. Overview
1. Accessibility
1. Examples

Author: I'd seen it while doing the change...

  <span class="visually-hidden">Helper</span>
</button>
```

> ## Accessibility

Reviewer: I'd add some blank lines around this title

For accessibility purpose, [content on hover or focus should be dismissible, hoverable and persistent](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).
With Boosted tooltips, no mechanism is available to dismiss the tooltip without moving pointer hover or keyboard focus (due to Popper library's limitations).

> So, developers should pay attention that the tooltip never hides an important content.

Reviewer: ```suggestion
Please pay attention that the tooltip never hides an important content.
```
